comps/dataprep/src/utils.py

```diff
@@ -319,47 +320,50 @@ def load_csv(input_path):
     return content_list
 
 
-def load_image(image_path):
+async def load_image(image_path):
     """Load the image file."""
     if os.getenv("SUMMARIZE_IMAGE_VIA_LVM", None) == "1":
         query = "Please summarize this image."
         image_b64_str = base64.b64encode(open(image_path, "rb").read()).decode()
-        response = requests.post(
-            "http://localhost:9399/v1/lvm",
-            data=json.dumps({"image": image_b64_str, "prompt": query}),
-            headers={"Content-Type": "application/json"},
-            proxies={"http": None},
-        )
-        return response.json()["text"].strip()
+        async with aiohttp.ClientSession() as session:
+            response = await session.post(
+                url="http://localhost:9399/v1/lvm",
+                data=json.dumps({"image": image_b64_str, "prompt": query}),
+                headers={"Content-Type": "application/json"},
+                proxy=None,
+            )
+
+            json_data = await response.json()
+        return json_data["text"].strip()
     loader = UnstructuredImageLoader(image_path)
     text = loader.load()[0].page_content
     return text.strip()
 
 
-def load_svg(svg_path):
+async def load_svg(svg_path):
     """Load the svg file."""
     png_path = svg_path.replace(".svg", ".png")
     cairosvg.svg2png(url=svg_path, write_to=png_path)
-    text = load_image(png_path)
+    text = await load_image(png_path)
     os.remove(png_path)
     return text
 
 
-def document_loader(doc_path):
+async def document_loader(doc_path):
```

Comment:
why no await for load_pdf, load_html etc?

Reply:
Seems these two functions don't contain synchronized time-cost operation such as "requests.post", thus don't need to add.